Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -266,6 +272,34 @@ void DMainWindow::OnPlay()
 	}
 }
 
+// Toggles fullscreen
+// Game assumed running
```

Comment:
The game assumed running is there because m_render_widget is not instantiated until a game is started. Running this code without a game runnign would be bad.